src/qt/askpassphrasedialog.h

```diff
@@ -6,6 +6,7 @@
 #define BITCOIN_QT_ASKPASSPHRASEDIALOG_H
 
 #include <QDialog>
+#include <QLineEdit>
```

Comment:
Move to .cpp and just forward declare QLineEdit.

Reply:
It is required in this file because of `QLineEdit` is a parameter of `updatePasswordStrengthIndicator`

Reply:
It's a pointer to QLineEdit, the compiler doesn't need the definition so a forward declaration is enough.

Reply:
I agree with @promag here, it's already done this way with `WalletModel` in the same files.

Reply:
But the `updatePasswordStrengthIndicator` needs access to the `ui` variable. I could give it over a parameter the current solution is more elegant IMO

Reply:
Right, so instead of this:
```cpp
this->updatePasswordStrengthIndicator(ui->passEdit2);
```
just do
```cpp
// receive password as const 
this->updatePasswordStrengthIndicator(ui->passEdit2->text());QString&
```
or 
```cpp
// access ui->passEdit2 directly
this->updatePasswordStrengthIndicator(); 
```
and in either cases you should add `#include <QLineEdit>` in src/qt/askpassphrasedialog.cpp.